docs/website/docs/dlt-ecosystem/verified-sources/airtable.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Airtable
+description: dlt verified source for Airtable
+keywords: [airtable api, airtable verified source, airtable]
+---
+
+# Airtable
+
+Airtable is a cloud-based platform that merges spreadsheet and database functionalities for easy
+data management and collaboration.
+
+This Airtable `dlt` verified source and
+[pipeline example](https://github.com/dlt-hub/verified-sources/blob/master/sources/airtable_pipeline.py)
+loads data using “Airtable API” to the destination of your choice.
+
+Sources and resources that can be loaded using this verified source are:
+
+| Name              | Description                                |
+| ----------------- | ------------------------------------------ |
+| airtable_source   | Retrieves tables from an airtable base     |
+| airtable_resource | Retrives data from a single airtable table |
```

Comment:
After this, I had found it mostly capitalized
```suggestion
| airtable_source   | Retrieves tables from an Airtable base     |
| airtable_resource | Retrives data from a single Airtable table |
```

Reply:
totally agree